cmd/canary/canary.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const externalMountPoint = "/data"
+
+type ping struct{}
+
+var _ http.Handler = &ping{}
+
+func (p *ping) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	str := fmt.Sprintf("%s/%s : %s. Hostname: %s", r.RemoteAddr, r.URL, r.UserAgent(), os.Getenv("HOSTNAME"))
+	// open existing file
+	f, err := os.OpenFile(path.Join(externalMountPoint, "lastrequest.log"), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
```

Comment:
This was not appending file. So if we add hostname it would be nice to see when the hostname changes in the log history.

Reply:
the hostname should never change in the log history ;)
don't append - we have no way of cleaning up the file